doc/cliques.xml

```diff
@@ -396,6 +444,9 @@ gap> DigraphMaximalIndependentSet(gr, [1], [], 3);
     <Ref Oper="IsMaximalIndependentSet"/> and <Ref
       Oper="IsIndependentSet"/>.<P/>
 
+    The returned list will be mutable if and only if the input is mutable. The
+    lists contained in the output list are always immutable.
```

Comment:
This behaviour is weird, I'm not sure if it is intentional or not, maybe this should be its own issue @le27 ?